tgui/packages/tgui/interfaces/NTRecruiter.js

```diff
@@ -80,8 +80,9 @@ export const NTRecruiter = (props, context) => {
           >
             <LabeledList>
               <LabeledListItem label="1#" color="silver">
-                To win this game you must hire/dismiss <b>{total_curriculums}</b> candidates,
-                one wrongly made choice leads to a game over.
+                To win this game you must hire/dismiss{' '}
```

Comment:
this seems sus, @Sirryan2002 is this okay?

Reply:
Very sus

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/12197162/dc755a99-b751-4c2a-b014-fd1915052b3d)

still works fine, so not sure why it is doing here

Reply:
Yes, this is fine

Reply:
its the same output, this is just something prettier does for some stupid reason, there's not a practical solution to it tbh